Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8856,17 +8856,21 @@ def __init__(self, response, continue_to_poll=False, args_for_next_run=None):
         :type args_for_next_run: ``Dict``
         :param args_for_next_run: The arguments to use in the next iteration. Will use the input args in case of None
 
+        :type partial_result: ``CommandResults``
+        :param partial_result: CommandResults to return, even though we will poll again
```

Comment:
I think `command_results` is a better name than `partial_results`. It's hard to understand how `partial_results` will act, while `command_results` will be very clear.

Reply:
partial_results I think makes it clear that the commandresults provided will be used in the case that we will continue to poll. Command_results I think can be mistaken for what I've named here response - the final result to provide